README.md

```diff
@@ -30,8 +30,39 @@ Uploading a pipeline is done via the `fly set-pipeline` command, which is includ
 
 See [the ZAP pipeline README](pipelines/zap/#production).
 
-## Feedback
+## Adding a Project
+
+The `targets.json` file in the `config` directory acts as a whitelist against the `team-api` list of projects.
+
+To get a new project added to the scans:
+
+1. Ensure that your project appears in the [team-api](https://team-api.18f.gov/api/projects/). The directions for doing that are [here](https://github.com/18F/team-api.18f.gov#adding-project-data).
+
+1. Submit a PR to this repo after adding an entry in `config/targets.json` like this:
+
+  ```
```

Comment:
Minor: indent the backticks two more spaces, and include `json` after these top ones for syntax highlighting.